README.md

```diff
@@ -12,7 +12,7 @@ This Terraform module deploys an S3-hosted static site with HTTPS enabled.
 ## Usage
 ```hcl
 module "s3_site" {
-  source    = "github.com/byu-oit/terraform-aws-s3staticsite?ref=v2.0.0"
+  source    = "github.com/byu-oit/terraform-aws-s3staticsite?ref=v2.0.1"
```

Comment:
you might need to add the new version to the example as well?

Reply:
@yoshutch Done.